internal/resources/policy/constants.go

```diff
@@ -6,9 +6,26 @@ SPDX-License-Identifier: MPL-2.0
 package policy
 
 const (
-	NamespaceSelectorKey = "namespace_selector"
-	MatchExpressionsKey  = "match_expressions"
-	KeyKey               = "key"
-	OperatorKey          = "operator"
-	ValuesKey            = "values"
+	NamespaceSelectorKey      = "namespace_selector"
+	MatchExpressionsKey       = "match_expressions"
+	KeyKey                    = "key"
+	OperatorKey               = "operator"
+	ValuesKey                 = "values"
+	ScopeKey                  = "scope"
+	clusterKey                = "cluster"
+	clusterGroupKey           = "cluster_group"
+	organizationKey           = "organization"
+	SpecKey                   = "spec"
+	NameKey                   = "name"
+	InputKey                  = "input"
+	RecipeVersionDefaultValue = "v1"
+	UnknownRecipe             = ""
```

Comment:
Why is empty string a constant ? and how it is being used. This seems out of place

Reply:
We are using this constant in the way: `unknownRecipe  recipe = policy.UnknownRecipe` for security and custom policy. Made this change as per [the comment](https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/81#discussion_r1011504893).